test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+CHANGED_FILES=$(git diff 7901137c1edf74d3f8c7c9fe1081e87cf29a4af2...30cfbab1672419d9f9d8dc123a5da07eb11eb0e9 --name-only)
```

Comment:
This is the head and tail hash for the O3DE repo, right?

Reply:
Oops, that test script shouldn't be in there. Deleted!